base/setup/usetup/mui.h

```diff
@@ -102,67 +102,66 @@ MUISetStyledText(
 /* MUI Text IDs */
 
 /* Static MUI Text */
-#define TEXT_ID_STATIC (-1)
+#define TEXT_ID_STATIC                        (-1)
```

Comment:
Please avoid reformatting the whole file for the moment (as it may collide with other mui.h changes in other PRs, and definitively collides with other changes I have in the GPT support improvement branches.)
Just do the change of adding or removing the string/page IDs , and without shifting the other values.